common/lib/xmodule/xmodule/editing_module.py

```diff
@@ -31,20 +31,24 @@ def non_editable_metadata_fields(self):
         """
         `data` should not be editable in the Studio settings editor.
         """
-        non_editable_fields = super(EditingDescriptor, self).non_editable_metadata_fields
+        non_editable_fields = super(EditingMixin, self).non_editable_metadata_fields
         non_editable_fields.append(self.fields['data'])
         return non_editable_fields
 
     # cdodge: a little refactoring here, since we're basically doing the same thing
     # here as with our parent class, let's call into it to get the basic fields
     # set and then add our additional fields. Trying to keep it DRY.
     def get_context(self):
-        _context = MakoModuleDescriptor.get_context(self)
+        _context = MakoTemplateBlockBase.get_context(self)
```

Comment:
`MakoTemplateBlockBase` where `get_context()` is implemented is superclass of `MakoModuleDescriptor`.